go/registry/api/api_test.go

```diff
@@ -137,104 +137,6 @@ func TestVerifyRegisterNodeArgs(t *testing.T) {
 				VRF: &node.VRFInfo{
 					ID: nodeVRFSigner.Public(),
 				},
-				Roles:      node.RoleConsensusRPC,
-				Expiration: 11,
-			},
-			ErrInvalidArgument,
-			"invalid consensus RPC node (missing TLS address)",
-		},
-		{
-			node.Node{
-				Versioned: cbor.NewVersioned(2),
-				ID:        nodeSigner.Public(),
-				EntityID:  entityID1,
-				Consensus: node.ConsensusInfo{
-					ID: nodeConsensusSigner.Public(),
-					Addresses: []node.ConsensusAddress{
-						{ID: nodeConsensusSigner.Public(), Address: node.Address{IP: net.IPv4(127, 0, 0, 1), Port: 9000}},
-					},
-				},
-				TLS: node.TLSInfo{
-					PubKey: nodeTLSSigner.Public(),
-					Addresses: []node.TLSAddress{
-						{
-							PubKey:  nodeTLSSigner.Public(),
-							Address: node.Address{IP: net.IPv4(127, 0, 0, 2), Port: 9001},
-						},
-					},
-				},
-				P2P: node.P2PInfo{
-					ID: nodeP2PSigner.Public(),
-				},
-				VRF: &node.VRFInfo{
-					ID: nodeVRFSigner.Public(),
-				},
-				Roles:      node.RoleConsensusRPC,
-				Expiration: 11,
-			},
-			ErrInvalidArgument,
-			"invalid consensus RPC node (missing P2P address)",
```

Comment:
Instead of removing this case, should we just use a different role? All nodes should now include P2P addresses anyway.

Reply:
Makes sense, done.